.env

```diff
@@ -19,3 +19,4 @@ RN_BRIDGE_THRESHOLD_WARNINGS=0
 RPC_NETWORKS_ONLY=0
 STICKERS_ENABLED=1
 PARTITIONED_TOPIC=1
+CHAOS_MODE=1
```

Comment:
I think it should be a dynamic option (in app-db, but we can default it from here)

Reply:
let's figure how we handle Etherscan disabling in this PR and I will file another one with a new profile setting which is stored in app-db

Reply:
sure!

Reply:
well, added in this PR :)